### DISCUSSION TITLE
Is Angular 15 officially supported ?

### DISCUSSION THREAD

**Discussion by ES-Six** [ngneat/transloco#613] [Q&A] [answered]
I read the documentation here : https://ngneat.github.io/transloco/docs/getting-started/installation#angular--transloco-versions

But angular 14 is not listed in the compatibility tab, I think angular 14 should be supported and the doc is just not up to date.

So I have some questions :
- Is Angular 15 supported by transloco ?
- Does anyone succeed to configure a working transloco + angular 15 project ?
- Is there incompatibility between Angular 15 and transloco latest ?

Thanks for reading.

**Comment by shaharkazaz** [comment, marked as answer]
@ES-Six Transloco should work as expected with Angular >=v13.
I'll update the docs, are you encountering any issues?

**Comment by ES-Six** [comment]
Thank you @shaharkazaz I've planed to upgrade from Angular 13 to Angular 15 soon. So I dont know if there are issues for know. I will let you know if there is any.

**Reply by ES-Six** [reply]
Update after testing :
Transloco work well with Angular 15, upgrade done flawlessly, no error, no breaking changes were found.

Tested with :
- Angular: 15.2.0
- @ngneat/transloco: 4.2.6
- @ngneat/transloco-messageformat: 4.1.0

Thank you :)

**Reply by hermanseder** [reply]
I think there exists a problem with “ngneat/transloco-keys-manager” in combination with Angular 15. If I use the new feature of “self-closing tags” for custom components, the tool ignores such templates and does not extract any keys. The problem is, as far as I found out, the template parser, because the tool uses the “old” Angular compiler which, of course, does not allow “self-closing tags”.

Is there an option to fix this problem with the current version or is an update of the packages in the “ngneat/transloco-keys-manager” necessary? The annoying point is, that the tool silently ignores files with the new syntax.

Thanks in advance.

I used the following versions:
•	@angular/compiler: 15.1.0
•	@ngneat/transloco": 4.2.6
•	@ngneat/transloco-keys-manager: 3.6.2

**Reply by shaharkazaz** [reply]
@hermanseder Transloco support v15.
There is a dedicated repo for the transloco keys manager, if there is an issue there you are welcome to open an issue and a PR 🙂
